database/entity/0075-add_updated_by_contribution/Contribution.ts

```diff
@@ -0,0 +1,104 @@
+import { Decimal } from 'decimal.js-light'
+import {
+  BaseEntity,
+  Column,
+  Entity,
+  PrimaryGeneratedColumn,
+  DeleteDateColumn,
+  JoinColumn,
+  ManyToOne,
+  OneToMany,
+  OneToOne,
+} from 'typeorm'
+import { DecimalTransformer } from '../../src/typeorm/DecimalTransformer'
+import { User } from '../User'
+import { ContributionMessage } from '../ContributionMessage'
+import { Transaction } from '../Transaction'
+
+@Entity('contributions')
+export class Contribution extends BaseEntity {
+  @PrimaryGeneratedColumn('increment', { unsigned: true })
+  id: number
+
+  @Column({ unsigned: true, nullable: false, name: 'user_id' })
+  userId: number
+
+  @ManyToOne(() => User, (user) => user.contributions)
+  @JoinColumn({ name: 'user_id' })
+  user: User
+
+  @Column({ type: 'datetime', default: () => 'CURRENT_TIMESTAMP', name: 'created_at' })
+  createdAt: Date
+
+  @Column({ type: 'datetime', nullable: false, name: 'contribution_date' })
+  contributionDate: Date
+
+  @Column({ length: 255, nullable: false, collation: 'utf8mb4_unicode_ci' })
+  memo: string
+
+  @Column({
+    type: 'decimal',
+    precision: 40,
+    scale: 20,
+    nullable: false,
+    transformer: DecimalTransformer,
+  })
+  amount: Decimal
+
+  @Column({ unsigned: true, nullable: true, name: 'moderator_id' })
+  moderatorId: number
+
+  @Column({ unsigned: true, nullable: true, name: 'contribution_link_id' })
+  contributionLinkId: number
+
+  @Column({ unsigned: true, nullable: true, name: 'confirmed_by' })
+  confirmedBy: number
+
+  @Column({ nullable: true, name: 'confirmed_at' })
+  confirmedAt: Date
```

Comment:
remember the ms-dimension of date-columns

Reply:
I have simply copied the previous entity and added only the field: updatedBy
Do you know if the current code use ms precision for one or all contribution date fields?
To decide if we could use @CreateDateColumn and/or @UpdateDateColumn here I must first check the current code how the fields are used and which problems it could make changing that. But this kind of refactoring wasn't my intention for this pr.